python_modules/dagster/dagster/_core/definitions/asset_dep.py

```diff
@@ -6,12 +6,17 @@
 from dagster._core.definitions.assets import AssetsDefinition
 from dagster._core.definitions.partition_mapping import PartitionMapping
 from dagster._core.definitions.source_asset import SourceAsset
+from dagster._core.errors import DagsterInvalidDefinitionError
 
 from .events import (
     AssetKey,
     CoercibleToAssetKey,
 )
 
+CoercibleToAssetDep = Union[
+    CoercibleToAssetKey, AssetSpec, AssetsDefinition, SourceAsset, "AssetDep"
+]
+
 
 @experimental
 class AssetDep(
```

Comment:
Change `AssetDep.asset_key` to `AssetDep.key` as per https://github.com/dagster-io/dagster/pull/16544?

Reply:
I don't think so at first blush. `key` could mean that the dependency _itself_ has a key.